emissionsapi/db.py

```diff
@@ -27,6 +27,14 @@
 Base = declarative_base()
 
 
+class File(Base):
+    """ORM Object for the nc files.
+    """
+    # Tablename
+    __tablename__ = 'file'
+    filename = Column(String, primary_key=True)
```

Comment:
what is the "primary_key=True" doing?

Reply:
These fields relate to database columns and `primary_key` means in this context that it's the column by which an entry can be identified, implying that it may not be null, nor may two identical entries exist

Reply:
ah ok. thanks for the explanation. 👍